superset/assets/javascripts/dashboard/actions/dashboard.js

```diff
@@ -0,0 +1,165 @@
+import $ from 'jquery';
```

Comment:
should this file be called `dashboardState`?

Reply:
fixed.